test/suites/standard1/configuration.robot

```diff
@@ -63,6 +75,39 @@ Config Flags Are Logged in Audit Flags
     Pattern Should Appear In Log Output    ${CURSOR}    FLAG: --audit-log-maxbackup=\"1000\"
     Pattern Should Appear In Log Output    ${CURSOR}    FLAG: --audit-log-maxage=\"1000\"
 
+Deploy MicroShift With LVMS By Default
```

Comment:
Nit: Can we be consistent in `LVMS` vs `LVMO` name usage? I think we should use `LVMS` in docs.